world-bank-app/src/components/TwoCountriesOneIndicatorOneYear.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, PureComponent } from "react";
+import Navbar from "react-bootstrap/Navbar";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import Network from "../Network.js";
+import Logo from "../assets/bank-logo.png";
+
+export default function TwoCountriesOneIndicatorOneYear(props) {
+  const network = new Network();
+  const [data, setData] = useState();
+
+  useEffect(() => {
+    (async () => {
+      setData(
+        await network.getTwoCountriesIndicatorYearInfo(
+          props.countries[0],
+          props.countries[1],
+          props.indicator,
+          props.year
+        )
+      );
+    })();
+  }, []);
+
+  function checkResponse() {
+    if ("response" in data) {
+      return (
+        <p>
+          Please choose a different first country, second country or indicator
+          combination.
+        </p>
+      );
+    } else {
+      return (
+        <Container className="py-5">
+          <h2>{props.year}</h2>
+          <h4>{props.indicator}</h4>
+          <h4>
+            {props.countries[0]}: {Number(data[0].Value)}
+          </h4>{" "}
+          <h4>
+            {props.countries[1]}: {Number(data[1].Value)}
+          </h4>
+        </Container>
+      );
+    }
+  }
+
+  return (
+    <Container className="py-4">
+      <header className="border-bottom pb-3 mb-4">
+        <Navbar>
+          <Container>
+            <Navbar.Brand href="#home">
```

Comment:
```suggestion
            <Navbar.Brand href="/">
```

Make this a / to allow logo navigation